client/src/ui/containers/useShoot.ts

```diff
@@ -1,18 +1,27 @@
 import { ViewGame, ViewBoard } from "controllers/model/index";
 import { gameController } from "config";
-
-//TODO alert the user when ship is destroyed
+import { useSnackbar } from "notistack";
+import { useStyles } from "ui/styles";
 
 export function useShoot(setGame: (game: ViewGame) => void, gameId: string) {
+  const { enqueueSnackbar } = useSnackbar();
+  const { classes } = useStyles();
+
   const handleCellClick = (x: number, y: number) => {
     gameController()
       .shoot(gameId, x, y)
       .subscribe((shotResult) => {
-        if (shotResult.ship?.destroyed) {
-          console.log(
-            "Ship destroyed of length: " + shotResult.ship.coordinates.length
+        if (shotResult.ship?.destroyed)
+          enqueueSnackbar(
+            "You destroyed a ship of length " +
+              shotResult.ship.coordinates.length,
+            {
+              autoHideDuration: 3000,
+              anchorOrigin: { horizontal: "right", vertical: "bottom" },
+              variant: "info",
+              className: `${classes.snackbar}`,
+            }
```

Comment:
pasirašyk wrapper hook, į kurį paduosi suformuotą message, variant ir className(jei čia ir Cell.tsx ne tokie patys style). To wrapper netestuok, o čia kur jį naudosi bus lengviau užmockinti